web/frontend/tslint.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
Errr, I just realized this. Shouldn't we be using eslint instead of tslint ? 
https://www.darraghoriordan.com/2020/03/06/upgrade-tslint-deprecated-to-eslint/